eng/scripts/automation_init.sh

```diff
@@ -1,25 +1,58 @@
+#!/bin/bash
+
+if [ -z $1 ]; then
+    echo "Please input outputfile"
+    exit 1
+fi
+echo $1
+
```

Comment:
As this init script will be used in current SDK automation, we may also change the config if `bash` is needed.